admin/scss/_actionTabs.scss

```diff
@@ -125,7 +125,7 @@ $border: 1px solid darken(#D9D9D9, 15%);
 					font-size: 12px;
 				}	
 
-				#Form_AddForm_PageType_Holder ul {
+				#Form_AddForm_PageType ul {
```

Comment:
We should not be, but I couldn't figure out why this selector was here. My suspicion is that if we remove it we'd find out soon enough why it was necessary, so I left it in place. :D